model_card_toolkit/version.py

```diff
@@ -11,7 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Contains the version string of the Model Card Toolkit."""
+"""Defines Model Card Toolkit version information."""
 
-# Note that setup.py uses this version.
-__version__ = '1.3.2'
+# We follow Semantic Versioning (https://semver.org/)
+_MAJOR_VERSION = "1"
+_MINOR_VERSION = "3"
+_PATCH_VERSION = "2"
```

Comment:
@codesue Since 1.3.2 already exists on Pypi, should we release 1.3.3 to Test PyPI and if it works, we push 1.4.0 to PyPI?